config/routes.rb

```diff
@@ -6,6 +6,7 @@
     # admin namespace is listed first intentionally
     namespace :admin do
       root to: 'posts#index'
+      post "posts/image" => 'posts#upload_image'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.